docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,544 @@
+
+To simplify the tutorial, we assume this additional orderer is part of the same organization as the previous three orderers and the orderer organization is already part of the channel configuration. If the organization of the new orderer is not already part of the channel configuration, a channel configuration update that adds the organization must be submitted (and signed by a set of signatures that satisfies the orderer group's admins policy) before the new orderer can begin pulling blocks. An orderer can pull blocks from other cluster members if its owning organization is defined in the channel configuration, and its orderer signature satisfies the `/Channel/Readers` policy.
+
+For this tutorial, the new orderer is not part of the consenter set. Run the following command to join the new orderer to the channel:
+
+```
+osnadmin channel join –-channel-id [CHANNEL_NAME]  --config-block [CHANNEL_CONFIG_BLOCK] -o [ORDERER_ADMIN_LISTENADDRESS] --ca-file [TLS_CA_ROOT_CERT] --client-cert [OSN_TLS_SIGN_CERT] --client-key [OSN_TLS_PRIVATE_KEY]
+```
+
+An orderer can join the channel by providing the genesis block, or the latest config block. But the value of `consensusRelation` will always be "follower" until this orderer is added to the channel's consenter set, by submitting an update to the channel configuration.
+
+If joining from the genesis block, the output of this command looks similar to:
+```
+{
+  "name": "channel1",
+  "url": "participation/v1/channels/channel1",
+  "status": "active",
+  "consensusRelation": "follower",
+  "height": 1
+}
+```
+
+Otherwise, if joining from the latest config block, the `status` is "onboarding". In that case, after the channel ledger has caught up to the specified config block, you can issue the same `osnadmin channel list` command to confirm the status changes to **active** and the ledger height has caught up. For example, if the total number of blocks on the channel ledger at the specified config block was `1151`, after the ledger catches up, the output would look similar to:
+
+```
+{
+  "name": "channel1",
+  "url": "participation/v1/channels/channel1",
+  "status": "active",
+  "consensusRelation": "follower",
+  "height": 1152
+}
```

Comment:
1151, actually; as the follower pulls the config-block that corresponds to the join-block given to it, it changes to "active". However, it is important to note that the transition from "onboarding" to "active" is not the only criteria used to decide whether it is the right time to add the new OSN to the channel's consenters set. What it more important is the `height` that the new OSN reached, compared to the height of the consenters. For example, the last config block may be at height 1151, but there might be a huge number of normal block after it.... After OSN4 becomes "active" (caught up to the last config), it is better to compare the height of OSN1 with that of OSN4, and if they are "close enough" - only then add OSN4 to the consenters set of the channel.